diff --git a/node_modules/playwright-core/lib/server/chromium/crBrowser.js b/node_modules/playwright-core/lib/server/chromium/crBrowser.js
index 4437688..f7754c5 100644
--- a/node_modules/playwright-core/lib/server/chromium/crBrowser.js
+++ b/node_modules/playwright-core/lib/server/chromium/crBrowser.js
@@ -354,7 +354,7 @@ class CRBrowserContext extends _browserContext.BrowserContext {
     (0, _utils.assert)(!Array.from(this._browser._crPages.values()).some(page => page._browserContext === this));
     const promises = [super._initialize()];
 
-    if (this._browser.options.name !== 'electron' && this._browser.options.name !== 'clank') {
+    if (this._browser.options.name !== 'electron' && this._browser.options.name !== 'clank' && !global.isDesktopApplication) {
       promises.push(this._browser._session.send('Browser.setDownloadBehavior', {
         behavior: this._options.acceptDownloads ? 'allowAndName' : 'deny',
         browserContextId: this._browserContextId,
